src/containers/Header/NetworkPicker/NetworkPicker.tsx

```diff
@@ -18,6 +18,7 @@ const STATIC_ENV_LINKS: Record<string, string | undefined> = {
   testnet: process.env.VITE_TESTNET_LINK,
   devnet: process.env.VITE_DEVNET_LINK,
   amm: process.env.VITE_AMM_LINK,
+  hooks: process.env.VITE_HOOKS_LINK,
```

Comment:
This should maybe be worded differently especially in light of the hooks sidechain